src/coreclr/vm/jitinterface.cpp

```diff
@@ -6058,8 +6061,7 @@ CorInfoHelpFunc CEEInfo::getCastingHelperStatic(TypeHandle clsHnd, bool fThrowin
             *pfClassMustBeRestored = true;
```

Comment:
The `pfClassMustBeRestored` argument is dead code and it can be deleted. It only controls whether we end up calling `classMustBeLoadedBeforeCodeIsRun` that is a no-op.